src/aggregator/services/budget-insight/budget-insight.utils.ts

```diff
@@ -25,18 +27,25 @@ import {
 export const mapBudgetInsightAccount = (
   accounts: BudgetInsightAccount[],
   connections: Connection[],
+  connectionsInfo: { [key: string]: BudgetInsightOwner },
 ): PostBanksUserAccountDTO[] =>
   accounts.map((acc: BudgetInsightAccount) => {
     const connection: Connection | undefined = connections.find((con) => con.id === acc.id_connection);
+    const ownerInfo: BudgetInsightOwner | undefined =
+      !isNil(connection) && !isNil(connection?.id) ? connectionsInfo[connection.id] : undefined;
```

Comment:
Since you've put the `?` operator, no need to double check

```suggestion
    const ownerInfo: BudgetInsightOwner | undefined =
      !isNil(connection?.id) ? connectionsInfo[connection.id] : undefined;
```

Reply:
It makes TypeScript fail, I don't know why exactly. (I can add `@ts-ignore` comment otherwise)
![Capture d’écran 2020-11-19 à 10 38 56](https://user-images.githubusercontent.com/9453090/99648579-7ba51280-2a53-11eb-8ad9-e784a8d3c292.png)
![Capture d’écran 2020-11-19 à 10 40 24](https://user-images.githubusercontent.com/9453090/99648730-a68f6680-2a53-11eb-8c1d-423d47349d4a.png)

Reply:
Same for `ownerInfo` below.

Reply:
Or I can do that.
```suggestion
    const ownerInfo: BudgetInsightOwner | undefined = get(connectionsInfo, `${connection?.id}`);
```

Reply:
OK, because of the `isNil` method! Try:

```suggestion
    const ownerInfo: BudgetInsightOwner | undefined =
      connection?.id !== undefined ? connectionsInfo[connection.id] : undefined;
```